doc/index.md

```diff
@@ -120,53 +105,43 @@ public class Server : BaseClass
 
 public class Connection 
 {
-    public async Task NotifyRemote() 
+    public async Task InvokeRemote(Stream stream) 
     {
         var target = new Server();
-        var streams = Nerdbank.FullDuplexStream.CreateStreams();
-        var clientRpc = JsonRpc.Attach(streams.Item1);
-        var serverRpc = JsonRpc.Attach(streams.Item2, target);
-        var myResult = await clientRpc.InvokeWithParameterObjectAsync<string>("test/InvokeTestMethod");
+        var rpc = JsonRpc.Attach(stream, target);
+        var myResult = await rpc.InvokeWithParameterObjectAsync<string>("test/InvokeTestMethod");
     }
 }
 ```
 
-## Crashing the process on exception
-In some cases, you may want to immediately crash the server process if certain exceptions are thrown. In this case, overriding the `IsFatalException` method will give you the desired functionality. Through `IsFatalException` you can access and respond to exceptions as they are observed.
+## Close stream on fatal errors
+In some cases, you may want to immediately close the streams if certain exceptions are thrown. In this case, overriding the `IsFatalException` method will give you the desired functionality. Through `IsFatalException` you can access and respond to exceptions as they are observed.
```

Comment:
On the subject of closing the stream, we should think about how this fits with the new `WebSocket` transport that we now support, where closing the connection involves sending an async message before actually disposing the connection. But as closing the stream on failure is not new functionality, reconciling these doesn't have to be in this PR, IMO.